### DISCUSSION TITLE
Additional localisation for form fields / validation

### DISCUSSION THREAD

**Discussion by bradleyw92** [solspace/craft-freeform#168] [Questions & Answers] [answered]
Noticed the project has: [en, nl, de] translations.

Is there any scope for other languages to be added such as Finland, Turkey, Italy, please?

**Comment by kjmartens** [comment, marked as answer]
Hi @bradleyw92, those are some of the translations that have been provided to us by the community. If you are willing to translate Freeform into other languages, we'd be happy to include them in Freeform. :)
